.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+name: test on CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ["2.7", "3.0", "3.1", "3.2", "head"]
```

The Travis build tested all the way back to Ruby 2.0. Backwards compatibility is a big deal for sshkit and so I wouldn't want to lose that coverage. Can you add those missing versions?

```suggestion
        ruby: ["2.0", "2.1", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7", "3.0", "3.1", "3.2", "head"]
```